bin/cli.js

```diff
@@ -78,8 +78,12 @@ For more information, see https://webpack.js.org/api/cli/.`);
 				if (moduleName === "webpack") {
 					errorMessage = `\n${moduleName} not installed`;
 					instructions = `Install webpack to start bundling: \u001b[32m\n  $ npm install --save-dev ${moduleName}\n`;
-
-					if (process.env.npm_execpath !== undefined && process.env.npm_execpath.includes("yarn")) {
+					const path = require("path");
```

Comment:
put them on top level for now. Bad for runtime perf but good for readability

Reply:
done 👍

Reply:
What about this? https://github.com/webpack/webpack-cli/pull/957#issuecomment-501775706